src/adagrams.js

```diff
@@ -1,8 +1,150 @@
 const Adagrams = {
   drawLetters() {
-    // Implement this method for wave 1
+    const allLetters = {
+  //pool of available letters and the qty they each have
+      A : 9,
+      B : 2, 
+      C : 2, 
+      D : 4, 
+      E : 12, 
+      F : 2, 
+      G : 3, 
+      H : 2, 
+      I : 9, 
+      J : 1, 
+      K : 1, 
+      L : 4, 
+      M : 2, 
+      N : 6, 
+      O : 8, 
+      P : 2, 
+      Q : 1, 
+      R : 6, 
+      S : 4, 
+      T : 6, 
+      U : 4, 
+      V : 2, 
+      W : 2, 
+      X : 1, 
+      Y : 2, 
+      Z : 1
+    };
+    
+    //wave 1
+    const letters = new Array();
+    for (let char in allLetters) {
+      for(let i = 0; i < allLetters[char]; i += 1) {
+        letters.push(char);
+      }
+    }
+
+    let hands = new Array();
+    for(let i = 0; i < 10; i += 1) {
+      const index = Math.floor(Math.random() * (letters.length));
+      hands.push(letters[index]);
+      letters.splice(index, 1);
```

Comment:
Good work removing the used letter from the `letters` array. Consider making a deep copy of `letters` so that you leave the pool of letters untouched.